Tasks/Common/VstsAzureHelpers_/VstsAzureHelpers_.psm1

```diff
@@ -51,5 +51,41 @@ function Initialize-Azure {
     }
 }
 
+function Get-AzureCmdletsVersion
+{
+    $module = Get-Module -Name AzureRM* -ListAvailable | Select-Object -First 1
+    if($module)
+    {
+        return ($module).Version
+    }
+    return (Get-Module Azure -ListAvailable).Version
+}
+
+function Get-AzureVersionComparison
```

Comment:
Also please do not add this function. I specifically deleted this function when I ported the Azure helpers module. It was a significant hinderance to readability. If you need to compare two version in PowerShell then just do: `([version]'1.2.3') -lt ([version]'4.5.6')`